.flake8

```diff
@@ -2,6 +2,7 @@
 ignore = 
     ### Disabled because of conflicts with black
     E203, # whitespace before ':'
+	F841, # local variable is assigned but not never used
```

Comment:
Write this instead:
```
### Disabled since making the samples look worse (or something like that)
    F841, # local variable is assigned but not never used
```
Notice the four spaces instead of a tab before F841